CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contribute to JaRIKo
+
+Feedback and pull requests are very welcome!
+
+## Ask for help, feature or report a bug
+
+Feel free to create feature request and bug report in the [issues](https://github.com/smeup/jariko/issues).
+Only make sure that your request/report is not the duplicate of an exiting issue.
+
+## Choose an issue to work on
+
+Issues marked [up-for-grabs](https://github.com/smeup/jariko/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22) 
+or [good first issue](https://github.com/smeup/jariko/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) 
+are supposed to be easy and isolated enough to be done by anyone having interest in contributing.
+
+## Build from source
+
+See [Development guide](docs/development.md)
+
+## Coding standard
+
+Follow the [Kotlin conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
+
+## Open a pull request
```

Comment:
On GitHub it is also possible to make instructions appear when the user create an issue and maybe also a PR

Reply:
I created `docs/PULL_REQUEST_TEMPLATE.md`: let's see how it works :-)